USAGE.md

```diff
@@ -4,7 +4,7 @@ Ebookmaker has to reliably make EPUB and MOBI for over 60,000 different titles e
 
 ## Crawling
 
-Ebookmaker starts with a document file path or URL, and the follows links and images to a depth determined by the `--max_depth` setting. It only follows links that are in the same directory or below; anything in the same directory linked by the will be included in the ebook it tries to build. The --noimages setting results in exclusion of images. If you don't want the ebook to include a resource that your HTML links to, use the `rel='nofollow'` attribute of the `a` tag.
+Ebookmaker starts with a document file path or URL, and the follows links and images to a depth determined by the `--max_depth` setting. It only follows links that are in the same directory or below; anything in the same directory linked by the will be included in the ebook it tries to build. The `*.noimages` filetyps exclude images. If you don't want the ebook to include a resource that your HTML links to, use the `rel='nofollow'` attribute of the `a` tag.
```

Comment:
> and the follows links

should maybe be?

> and then follows links

---

> linked by the will be

Is there a word missing between "the" and "will"?

---

filetyps --> filetypes? Should it be "filetype excludes" or "filetypes exclude"?

Reply:
Thanks, @srjfoo !

Reply:
You're welcome :)